Revit_Core_Engine/Create/View/View3D.cs

```diff
@@ -65,33 +72,28 @@ public static View View3D(this Document document, string viewName = null, Boundi
 
             if (viewTemplateId != null)
             {
+                if (!(document.GetElement(viewTemplateId) as View).IsTemplate)
```

Comment:
This will crash if there is no element under `viewTemplateId`.

```suggestion
                if ((document.GetElement(viewTemplateId) as View)?.IsTemplate != true)
```

...also not sure why did you move this part out of the try/catch clause if you are raising same warning for both cases anyways.

Reply:
But isn't it worth dropping a more specific error then? Now you only avoid one `catch`, but that does not change anything else than that, right?